main.py

```
@@ -20,78 +20,108 @@
+
+def handle_lobby(gamestate_has_changed: bool):
+    if gamestate_has_changed:
+        lobby.start_queue()
+        lobby.reset_after_dodge()
+
+def handle_readycheck(gamestate_has_changed: bool):
+    if gamestate_has_changed:
+        connection.update_primary_role()
+        lobby.reset_after_dodge()
+        lobby.accept_match()
+        champselect_loop_iteration = 1
```

Ruff (0.13.1)

Local variable `champselect_loop_iteration` is assigned to but never used

Remove assignment to unused variable `champselect_loop_iteration`

(F841)